Projects/NUCLEO-F401RE/Gate/lib/Gate/gate.c

```diff
@@ -220,7 +224,8 @@ static int serial_write(char *data, int len)
 {
     for (unsigned short i = 0; i < len; i++)
     {
-        while (!LL_USART_IsActiveFlag_TXE(USART2));
+        while (!LL_USART_IsActiveFlag_TXE(USART2))
+            ;
```

Comment:
Well, it's not very pretty but it seems there are no solutions...

Reply:
yes, using clang-format we can't do it differently...